style.css

```diff
@@ -83,3 +82,322 @@ h2 {
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.Feat-text {
+  width: 327px;
+  height: 147px;
+  top: 48px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin-bottom: 15px;
+  text-align: justify;
+}
+
+.about-section .btn {
+  align-self: flex-start;
+  margin-left: 80px;
+}
+
+.my-works {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.my-works h2 {
+  width: 327px;
+  border-bottom: 1px solid #606163;
+  height: 83px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.works-section-a {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding: 20px;
+}
+
+.projects {
+  background-image: url(src/Img\ Placeholder-proj.svg);
+  background-repeat: no-repeat;
+  text-align: center;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  background-position: center;
+  background-size: contain;
+  position: relative;
+}
+
+.projects h3 {
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 8px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  z-index: 3;
+}
+
+.projects .Feat-text {
+  width: 260px;
+  left: 16px;
+  top: 101px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 22px;
+  color: #fff;
+  margin: 1px 10px;
+  z-index: 2;
+}
+
+.proj-container {
+  display: grid;
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.about-section {
+  background-image: url(src/Group.svg);
+  background-repeat: no-repeat;
+  background-color: #1c1a19;
+  background-position: bottom right;
+  width: 100vw;
+  height: 100vh;
+  top: 4137px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+.about-section h2 {
+  color: #ff6b00;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+}
+
+.about-section p {
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
```

Comment:
- [ ] Please look out for common element styles because it is a bad practice if we repeat them for every element. I suggest that they are grouped under a class so they can be added to the elements that require them. This helps us to conform to the DRY principle.